ui/ui-frontend/projects/archive-search/src/app/archive/criteria-search/criteria-search.component.html

```diff
@@ -16,7 +16,7 @@
                   : criteriaValue.valueTranslated
                     ? ('ARCHIVE_SEARCH.SEARCH_CRITERIA_FILTER.' + getCategoryName(criteriaVal.category) + '.' + criteriaValue.label
                       | translate)
-                    : criteriaValue.label
+                    : (getCriteriaLabel(criteriaVal.key, criteriaValue) | truncate: 50)
```

Comment:
Dans le cas DATE, on n'utilise pas getCriteriaLabel sur les autres branches

```suggestion
                    : criteriaValue.label
```